client/components/share/facebook-share-preview/index.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+export class FacebookSharePreview extends PureComponent {
+
+	static PropTypes = {
+		articleUrl: PropTypes.string,
+		externalProfilePicture: PropTypes.string,
+		externalProfileUrl: PropTypes.string,
+		externalName: PropTypes.string,
+		imageUrl: PropTypes.string,
+		message: PropTypes.string,
+	};
+
+	render() {
+		const {
+			articleUrl,
+			externalProfilePicture,
+			externalProfileUrl,
+			externalName,
+			imageUrl,
+			message,
+			translate
+		} = this.props;
+		return (
+			<div className="facebook-share-preview">
+				<div className="facebook-share-preview__content">
+					<div className="facebook-share-preview__header">
+						<div className="facebook-share-preview__profile-picture-part">
+							<img
+								className="facebook-share-preview__profile-picture"
+								src={ externalProfilePicture }
+							/>
+						</div>
+						<div className="facebook-share-preview__profile-line-part">
+							<div className="facebook-share-preview__profile-line">
+								<a className="facebook-share-preview__profile-name" href={ externalProfileUrl }>
+									{ externalName }
+								</a>
+								<span>
+									{
+										translate( 'published an article on {{a}}WordPress{{/a}}', {
```

Comment:
@jhnstn - concatenating string is problematic for translation.
Can you please fi this so that the profile name is passed do the translate function as a component?
i.e `%name% publishd an article on {{a}}WordPress{{/a}}`